### PR TITLE
Added localisation for the language of user

### PR DESCRIPTION
**Pull request by ashish0910**
Currently language of user is not translated and is shown in english only . This PR added translation for the language of user .

Current behaviour:

Screenshot 2019-03-25 at 7 23 26 PM

Expected behaviour:

Screenshot 2019-03-25 at 7 22 24 PM

### CODE REVIEW

File: dashboard/public/l10n/locales.ini [hunk 1]
```diff
@@ -505,6 +535,16 @@ neighborhood_view=Vue voisinage
 journal_view=Journal
 WebApp=Web App
 App=App
+english=Anglais
+spanish=Espanol
+french=français
+german=allemand
```
Comment (NikhilM98) [inline dashboard/public/l10n/locales.ini]: Capitalize the first character of the language name.

File: dashboard/views/users.ejs [hunk 2]
```diff
@@ -100,16 +100,16 @@
...
-                                              <% if(data.users[i].language =='es') {%> <td>Spanish</td> <%} %>
-                                              <% if(data.users[i].language =='fr') {%> <td>French</td> <%} %>
-                                              <% if(data.users[i].language =='de') {%> <td>German</td> <%} %>
-                                              <% if(data.users[i].language =='pt') {%> <td>Portuguese</td> <%} %>
-                                              <% if(data.users[i].language =='ar') {%> <td>Arabic</td> <%} %>
-                                              <% if(data.users[i].language =='ja') {%> <td>Japanese</td> <%} %>
-                                              <% if(data.users[i].language =='pl') {%> <td>Polish</td> <%} %>
-                                              <% if(data.users[i].language =='ibo') {%> <td>Igbo</td> <%} %>
-                                              <% if(data.users[i].language =='yor') {%> <td>Yoruba</td> <%} %>
+                                              <% if(data.users[i].language =='en') {%> <td data-l10n-id="english"></td> <%} %>
```
Comment (NikhilM98) [inline dashboard/views/users.ejs]: Language while creating user also needs to be translated.

**Comment by ashish0910** [comment]
> Apart from the language change in users view. I wonder if the double translation can be implemented in the dropdown following Sugarizer [#293](https://github.com/llaske/sugarizer/pull/293).

Yes , check #146 for this

**Comment by ashish0910** [comment]
> Apart from the language change in users view. I wonder if the double translation can be implemented in the dropdown following Sugarizer [#293](https://github.com/llaske/sugarizer/pull/293).
done in #146

**Pull request merged**